astropy/io/ascii/cds.py

```diff
@@ -115,8 +127,12 @@ def get_cols(self, lines):
             match = re_col_def.match(line)
             if match:
                 col = core.Column(name=match.group('name'))
-                col.start = int(re.sub(r'[-\s]', '',
-                                       match.group('start') or match.group('end'))) - 1
+                col.start = (
+                    int(
+                        re.sub(r'[-\s]', '', match.group('start') or match.group('end'))
+                    )
+                    - 1
+                )
```

Comment:
I know we are not meant to quarrel excessively with the way the formatting rules are imposed by black, because, they are the rules, but this, to me personally, is way beyond the border into the horrendous (a number more, similar examples in `core.py` below).
Do we already have some standards at what point we may or should apply `# fmt:` decorators according to https://github.com/astropy/astropy/pull/13949#discussion_r1010780782 to protect such constructs?

Reply:
Yes anything you feel is horrendous warrants a `# fmt:` decorator. Please feel free to push commit(s) adding `# fmt:` and undoing `black` formatting as you feel fit.

Reply:
Reverting back here is fine, but I also stopped and looked at that line. IMO both versions are awkward but the Black version is better. In the original you have 3 nested parens and the end (always a bad sign) and a break after 2 of the 3 args to `re.sub()` which takes a lot of time to parse for me.  In the black version at least it is very easy to see that you have `int( some re.sub expression) - 1` (even if it does look weird).

Anyway, we knew this would happen, not a problem. :smile:

Reply:
I agree about the break in the middle of the args, though I hardly find it any easier to figure out where the `-1` belongs to in the black version with the 4th pair of `()` added!
Perhaps my alternative suggestion from https://github.com/astropy/astropy/pull/13956#discussion_r1012341124 would have been better, even if I'm not happy with the indentation level.

Yes, good to discuss some of these cases early on. 😁